tools/generation/lib/template.py

```diff
@@ -154,7 +154,7 @@ def _frontmatter(self, case_filename, case_values):
         features += self.attribs['meta'].get('features', [])
         features = list(OrderedDict.fromkeys(features))
         if len(features):
-            lines += ['features: ' + yaml.dump(features).strip()]
+            lines += ['features: ' + re.sub('\n\s*', ' ', yaml.dump(features).strip())]
```

Comment:
As a side note, we should probably do the same thing for all the other stuff that outputs `foo: [bar, baz]`, i.e. `flags` and `includes` (just a few lines below) and possibly others.